src/js/common/components/NavButtons.jsx

```diff
@@ -81,7 +81,7 @@ export default class NavButtons extends React.Component {
     );
 
     let buttons;
-    if (path.endsWith('review-and-submit')) {
+    if (path.match(/review-and-submit(\/)?$/)) {
```

Comment:
Nah, I'd just leave it with the `.match()`. 😄 This was mostly just my idle curiosity since the difference will be a fraction of a millisecond most of the time, methinks.